charmhelpers/contrib/storage/linux/ceph.py

```diff
@@ -1695,14 +1698,16 @@ def __init__(self, api_version=1, request_id=None, raw_request_data=None):
         if raw_request_data:
             request_data = json.loads(raw_request_data)
             self.api_version = request_data['api-version']
-            self.request_id = request_data['request-id']
             self.set_ops(request_data['ops'])
+            self.request_id = request_data['request-id']
         else:
             self.api_version = api_version
             if request_id:
                 self.request_id = request_id
             else:
-                self.request_id = str(uuid.uuid1())
+                # The below hash is the result of running
+                # `hashlib.sha1('[]'.encode()).hexdigest()`
+                self.request_id = '97d170e1550eee4afc0af065b78cda302a97674c'
```

Comment:
Personally, I'd move the value to a top-level declaration. Makes it easier to change and mock, IMO.